src/app/settings/WatchList.js

```diff
@@ -154,15 +154,16 @@ export default class WatchList extends React.Component {
     this.modifyEditorSite(site => site.frames.splice(index, 1));
   }
 
-  addToWatchList({domain, protocol}) {
+  addToWatchList({domain, protocol, port}) {
     if (domain.startsWith('www.')) {
       domain = domain.slice(4);
     }
+    const hostPort = `${domain}${port ? `:${port}` : ''}`;
```

Comment:
I find the naming `hostPort` unfortunate. You want to express "host + port", but it reads more like "the port of the host".

Reply:
Made it `hostAndPort`